src/Cache/Cache.php

```diff
@@ -100,9 +100,9 @@ class Cache
     /**
      * Cache Registry used for creating and using cache adapters.
      *
-     * @var \Cake\Cache\CacheRegistry
+     * @var \Cake\Cache\CacheRegistry|null
      */
-    protected static CacheRegistry $_registry;
+    protected static ?CacheRegistry $_registry;
```

Comment:
```suggestion
    protected static ?CacheRegistry $_registry = null;
```
is probably cleaner, right?